apps/custom-ui-app-docusign/public/js/panel.js

```diff
@@ -0,0 +1,62 @@
+// Shows a Snack Bar with a message
+async function showSnackBar(message) {
+    const sdk = await new AppExtensionsSdk().initialize();
+    await sdk.execute(AppExtensionsSdk.Command.SHOW_SNACKBAR, {
+        message
+    });
+}
+
+// Shows a modal with a given data
+async function showModal(data) {
+    const sdk = await new AppExtensionsSdk().initialize();
+    await sdk.execute(AppExtensionsSdk.Command.OPEN_MODAL, {
+        type: AppExtensionsSdk.Modal.JSON_MODAL,
+        action_id: '5c5fb865-f967-4bc2-88eb-96cae0ee74e7', // ID associated with the Custom UI Modal (visible in Marketplace Manager)
```

Comment:
Could this be moved to some config file and in the setup instructions mention that this ID has to be the one you get from Marketplace Manager

Reply:
Good point. Shall I switch to using the action name rather than a seperate config file? That way it becomes much more intuitive and doesn't require creating new file 🤔

Reply:
yes, action name is even better, but then during setup instructions you have to mention to create JSON modal with that specific name

Reply:
Done :) added a note in Readme as well